npm/private/test/snapshots/bzlmod/npm_defs.bzl

```diff
@@ -2048,371 +2048,369 @@ def npm_link_all_packages(name = "node_modules", imported_links = []):
     if link:
         if bazel_package == "js/private/worker/src":
             link_2(name = "{}/abortcontroller-polyfill".format(name))
-            link_targets.append("//{}:{}/abortcontroller-polyfill".format(bazel_package, name))
+            link_targets.append("//{}:{}/abortcontroller-polyfill".format(bazel_package, name)) if (not prod and not dev) or (prod and not True) or (dev and True) else None
```

Comment:
This generated `if ...` is ugly and I'm pondering how else we can do it. I think we can generate something simpler.

Can't we just compute a few vars at the top of the method such as `include_dev` and `include_prod`? Then this will be `if include_dev else None` or `if include_prod else None` or blank (or `if True else None` instead of blank).

Reply:
Yeah, something like that should be possible I think.

Reply:
After taking another look, I don't think it is possible. The contents of this fragment depend on whether the package in question is a production or dev dependency for the `bazel_package` in question. I.e., it's not a simple `include_dev` / `include_prod` boolean for each package, but rather a matrix of `bazel_package` x `link_alias`, which we're encoding as a nested conditional.

There might be some other way to structure this, but it would require a more invasive change.